File: service/src/main/java/com/pauldaniv/template/controller/TestController.java
package com.pauldaniv.template.controller;

import com.pauldaniv.template.api.TestOneService;
import com.pauldaniv.template.lib.second.consume.User;
import com.pauldaniv.template.request.TestOne;
import groovy.util.logging.Slf4j;
import lombok.RequiredArgsConstructor;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Service;

@Slf4j
@Service
@RequiredArgsConstructor
public class TestController {
    private final TestOneService testOneService;

    ResponseEntity<TestOne> getIt() {
        return ResponseEntity.ok(testOneService.getIt());
    }

    ResponseEntity<User> getUser() {
        return ResponseEntity.ok(new User("Test", "aoeu"));
    }
}

File: service/src/main/kotlin/com/pauldaniv/template/controller/TestController.kt
package com.pauldaniv.template.controller

import com.pauldaniv.template.api.TestOneService
import com.pauldaniv.template.lib.second.consume.User
import com.pauldaniv.template.request.TestOne
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import java.time.LocalDateTime

@RestController
@RequestMapping("/api")
class TestController(private val testOneService: TestOneService) {

  @GetMapping
  fun getIt(): ResponseEntity<TestOne> {
    return ResponseEntity.ok(testOneService.getIt())
  }

  @GetMapping("/example")
  fun getExamples(): ResponseEntity<User> {
    return ResponseEntity.ok(User("Test", LocalDateTime.now().toString()))
  }
}
